### ISSUE TITLE
-bash: vidlooper: command not found

### ISSUE THREAD

**Issue by bray12345giff** [alubbock/rpi-vidlooper#6]
Everything is working as expected, except for running vidlooper at login. I have added the vidlooper command to the end of bash.rc, but this is printed: -bash: vidlooper: command not found. I can run the command manually and it works fine. I am wondering if it has something to do with the PATH? I did have to update the path after I installed vidlooper as it wasn't in the PATH, but I'm not sure if that would have any effect

**Comment by bray12345giff** [comment]
@alubbock Any advice?

**Comment by bray12345giff** [comment]
I have tried multiple different ways of automatically starting vidlooper, and they all return the same thing as above. Not sure if it's a problem with the python app being installed in .local or what

**Comment by alubbock** [comment]
Yes, this is probably due to a `PATH` issue. After you `pip3 install rpi-vidlooper`, you can check where `vidlooper` is located by running `which vidlooper`. If that command returns nothing, try reinstalling/upgrading vidlooper using `pip3 install --upgrade vidlooper`.

If that `which` command does return a path (for me it's `/usr/local/bin/vidlooper`), then you could use that full path in your `.bashrc` in place of the `vidlooper` command, or add it to `PATH` in the same file.
